src/components/tooltip/__tests__/Tooltip.test.tsx

```diff
@@ -256,6 +256,29 @@ describe('components/tooltip/Tooltip', () => {
         });
     });
 
+    describe.each([[true], [false]])('with shouldStopEventPropagation being %o', shouldStopEventPropagation => {
+        describe.each([['click', 'keypress']])('simulate event %o', event => {
+            test('should stop event propagation as configured', () => {
+                const wrapper = shallow(
+                    <Tooltip isShown text="hi" shouldStopEventPropagation={shouldStopEventPropagation}>
+                        <button />
+                    </Tooltip>,
+                );
+                const instance = wrapper.instance();
+                const stop = jest.fn();
+                const nativeStop = jest.fn();
+                wrapper.find('div').simulate(event, {
```

Comment:
We tend not to use the simulate method and call the function directly.

Reply:
Hi Mike, do you have an alternative? what is the other way we do to test this?

Reply:
It is Mick, you can call the function directly on the instance.

Reply:
done